packages/terra-slider/src/Slider.jsx

```diff
@@ -0,0 +1,110 @@
+import React, { useEffect, useRef } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import ThemeContext from 'terra-theme-context';
+import styles from './Slider.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+  * Slider label value.
+  */
+  labelName: PropTypes.string.isRequired,
+
+  /**
+  * Minimum value in the range.
+  */
+  minimumValue: PropTypes.number.isRequired,
+
+  /**
+  * Maximum value in the range.
+  */
+  maximumValue: PropTypes.number.isRequired,
+
+  /**
+  * Default or initial value for the slider.
+  */
+  defaultValue: PropTypes.number.isRequired,
+
+  /**
+  * When true, will disable the field.
+  */
+  isDisabled: PropTypes.bool,
+
+  /**
+  * Minimum value label to display on the left side of the slider.
+  */
+  minimumLabel: PropTypes.string,
+
+  /**
+  * Maximum value label to display on the right side of the slider.
+  */
+  maximumLabel: PropTypes.string,
```

Comment:
Do we need a minimum label and maximum label?  Isn't the requirement to show the actual values?  We already have props for the minimum and maximum value.

Reply:
It was agreed that the minimum labels and maximum labels would be optional ways to display customized or more readable labels. These are needed to achieve that.